packages/stores/src/mockStore.ts

```diff
@@ -212,6 +212,11 @@ const mock = (): TStores & { is_mock: boolean } => {
                 redirectOnClick: jest.fn(),
                 setError: jest.fn(),
             },
+            services_error: {
+                code: '',
+                message: '',
+                type: '',
+            },
```

Comment:
```suggestion
            services_error: {},
```
https://github.com/binary-com/deriv-app/blob/master/packages/core/src/Stores/common-store.js#L75

Reply:
Thank you, noted